packages/connection-model/lib/model.js

```diff
@@ -805,11 +817,42 @@ Connection = AmpersandModel.extend({
     }
 
     if (attrs.authStrategy === 'KERBEROS') {
-      if (!attrs.kerberosServiceName) {
-        attrs.kerberosServiceName = KERBEROS_SERVICE_NAME_DEFAULT;
-      }
+      const authProperties = attrs.authMechanismProperties || {};
+      this.kerberosServiceName = defaultTo(
```

Comment:
is there a way to use plain javascript to do the same?

Reply:
wait isn't this just:

```
this.kerberosServiceName = attrs.kerberosServiceName || attrs.gssapiServiceName || authProperties.SERVICE_NAME;
```

Reply:
Kind of... in the end I want `??` here but can't use that in JS there. I don't want the ORs because I really want the first not `undefined` value.

Reply:
> I don't want the ORs because I really want the first not undefined value.

What is the difference?